build.gradle

```diff
@@ -11,6 +11,9 @@ allprojects {
     ////////////////////////////////////////////////////////////////////////////////////////////////////
     apply from:  rootProject.file("versions.gradle")
 
+    group = "com.r3.conclave"
```

Comment:
What is the benefit of moving the configuration outside the publishing scope if the result is the same?

Reply:
The Publication tasks default to the Project groupID and version values, and it's normal to set these values at the Project level.